pkg/k8shandler/logforwardingtopology/normalizer/collector_config.go

```diff
@@ -0,0 +1,83 @@
+package normalizer
+
+const (
+	fluentbitConf = `
+[SERVICE]
+    Log_Level info
+    Parsers_file /etc/fluent-bit/parsers.conf
+
+[INPUT]
+    Name tail
+    Path /var/log/containers/*_openshift*_*.log, /var/log/containers/*_kube*_*.log, /var/log/containers/*_default_*.log
+    Path_Key filename
+    Parser containerd
+    Exclude_Path /var/log/containers/collector*_openshift*_*.log, /var/log/containers/elasticsearch*_openshift*_*.log,/var/log/containers/kibana*_openshift*_*.log,/var/log/containers/normalizer*_openshift*_*.log
+    Tag kubernetes.*
+    DB /var/log/infra-containers.pos.db
+    Refresh_Interval 5
+
+[INPUT]
+    Name tail
+    Path /var/log/containers/*.log
+    Path_Key filename
+    Parser containerd
+    Exclude_Path /var/log/containers/*_openshift*_*.log, /var/log/containers/*_kube*_*.log, /var/log/containers/*_default_*.log
+    Tag kubernetes.*
```

Comment:
Same as above but this time, should we tag these as `container` or `app`? Not sure if it is desired to have two different inputs but the tags are exactly the same. WDYT?